diff -urN radiusd-cistron-1.6.6.org/src/Make.inc radiusd-cistron-1.6.6/src/Make.inc
--- radiusd-cistron-1.6.6.org/src/Make.inc	Sat Aug 10 02:18:10 2002
+++ radiusd-cistron-1.6.6/src/Make.inc	Sat Aug 10 02:19:53 2002
@@ -118,41 +118,41 @@
 
 install:
 	# Install the binaries
-	$(DINSTALL) $(SBINDIR)
-	$(DINSTALL) $(BINDIR)
-	$(DINSTALL) $(RADIUS_DIR)
-	$(INSTALL) -m 755 -s radiusd  $(SBINDIR)/radiusd
-	$(INSTALL) -m 755 -s radrelay $(SBINDIR)/radrelay
-	$(INSTALL) -m 755 -s radwho  $(BINDIR)
-	$(INSTALL) -m 755 -s raduse  $(BINDIR)
-	$(INSTALL) -m 755 -s radzap  $(BINDIR)
-	$(INSTALL) -m 755 -s radclient $(BINDIR)
-	test -z "$(DBM)" || $(INSTALL) -m 755 -s builddbm $(BINDIR)
-	$(INSTALL) -m 755    checkrad.pl $(SBINDIR)/checkrad
-	$(INSTALL) -m 755    radlast $(BINDIR)
-	$(INSTALL) -m 755    radtest $(BINDIR)
+	$(DINSTALL) $(DESTDIR)$(SBINDIR)
+	$(DINSTALL) $(DESTDIR)$(BINDIR)
+	$(DINSTALL) $(DESTDIR)$(RADIUS_DIR)
+	$(INSTALL) -m 755 -s radiusd  $(DESTDIR)$(SBINDIR)/radiusd
+	$(INSTALL) -m 755 -s radrelay $(DESTDIR)$(SBINDIR)/radrelay
+	$(INSTALL) -m 755 -s radwho  $(DESTDIR)$(BINDIR)
+	$(INSTALL) -m 755 -s raduse  $(DESTDIR)$(BINDIR)
+	$(INSTALL) -m 755 -s radzap  $(DESTDIR)$(BINDIR)
+	$(INSTALL) -m 755 -s radclient $(DESTDIR)$(BINDIR)
+	test -z "$(DBM)" || $(INSTALL) -m 755 -s builddbm $(DESTDIR)$(BINDIR)
+	$(INSTALL) -m 755    checkrad.pl $(DESTDIR)$(SBINDIR)/checkrad
+	$(INSTALL) -m 755    radlast $(DESTDIR)$(BINDIR)
+	$(INSTALL) -m 755    radtest $(DESTDIR)$(BINDIR)
 	# Install the man pages
 	@for section in 1 5 8; do \
 		$(DINSTALL) $(MANDIR)/man$$section; \
 		for manfile in ../doc/*.$$section*; do \
-		    echo $(INSTALL) -m 644 $$manfile $(MANDIR)/man$$section; \
-		    $(INSTALL) -m 644 $$manfile $(MANDIR)/man$$section; \
+		    echo $(INSTALL) -m 644 $$manfile $(DESTDIR)$(MANDIR)/man$$section; \
+		    $(INSTALL) -m 644 $$manfile $(DESTDIR)$(MANDIR)/man$$section; \
 		done ; \
 	done
-	test -n "$(DBM)" || rm -f $(MANDIR)/man8/builddbm.8rad
-	@if [ -d /etc/pam.d/ ] && [ ! -f /etc/pam.d/radius ]; then \
-		$(INSTALL) -m 644 ../redhat/radiusd-pam /etc/pam.d/radius;\
+	test -n "$(DBM)" || rm -f $(DESTDIR)$(MANDIR)/man8/builddbm.8rad
+	@if [ -d $(DESTDIR)/etc/pam.d/ ] && [ ! -f $(DESTDIR)/etc/pam.d/radius ]; then \
+		-$(INSTALL) -m 644 ../redhat/radiusd-pam /etc/pam.d/radius;\
 	fi
 	# Install the config files
 	@echo; echo "Creating/updating files in $(RADIUS_DIR)"; \
 	cd ../raddb; \
 	for i in [a-c]* [e-z]*; do \
-		[ ! -f $(RADIUS_DIR)/$$i ] && cp $$i $(RADIUS_DIR)/$$i; \
+		[ ! -f $(DESTDIR)$(RADIUS_DIR)/$$i ] && cp $$i $(DESTDIR)$(RADIUS_DIR)/$$i; \
 	done; \
 	for i in dictionary*; do \
-		[ ! -f $(RADIUS_DIR)/$$i ] && cp $$i $(RADIUS_DIR)/$$i; \
-		if [ "`find $$i -newer $(RADIUS_DIR)/$$i`" ]; then \
-			echo "** $(RADIUS_DIR)/$$i"; \
+		[ ! -f $(RADIUS_DIR)/$$i ] && cp $$i $(DESTDIR)$(RADIUS_DIR)/$$i; \
+		if [ "`find $$i -newer $(DESTDIR)$(RADIUS_DIR)/$$i`" ]; then \
+			echo "** $(DESTDIR)$(RADIUS_DIR)/$$i"; \
 			nt=1; \
 		fi; \
 	done; \
